content/docs/06-advanced/02-why-generative-ui.mdx

```diff
@@ -1,17 +1,24 @@
 ---
-title: Generative User Interfaces
-description: Overview of the AI and UI states
+title: Why Generative User Interfaces
+description: Why use generative user interfaces?
 ---
 
 # Generative User Interfaces
 
-Since language models can render user interfaces as part of their generations, the resulting model generations are referred to as generative user interfaces.
+Generative user interfaces elevate language models beyond plaintext and markdown. By rendering dynamic, adaptive user interfaces, language models can proactively adapt to the needs of the user.
+
```

Comment:
@nicoalbanese after moving this section to Advanced, I also removed the following copy, which I think we should move into the "Streaming React Components" section you're working on, wdyt?

```
Built for use with React Server Components, `ai/rsc` is optimized for delivering highly-dynamic, personalized software like an e-commerce store, a web app, a social network -- and the most dynamic interfaces of them all: generative user interfaces. The benefits of using React Server Components for generative user interfaces include:

- Reduced client bundle sizes
- Fewer round trips between the server and the client
- Improved developer experience around managing backend and frontend code
```